content/themes/chappress/style.css

```diff
@@ -122,8 +122,4 @@ a {
   font-size: 1rem;
   line-height: 20px; }
 
-.chapman-logo-link {
-  background-image: url("https://chappress-staging.chapman.edu/wp-content/themes/chappress/assets/images/cu-logo.png");
-  background-size: 100%; }
-
```

Comment:
Are these deletions not needed? Logo is still present?

Reply:
The login logo is active and is fine. I was trying to customize the home page logo but having issues with the customizing the hook into that.